app/controllers/api/v1/sessions_controller.rb

```diff
@@ -0,0 +1,32 @@
+class Api::V1::SessionsController < ApplicationController
+    def new 
```

Comment:
Add spec to test for login and logout functionality as well as to make sure that user can only login with proper credentials.